src/client/sandbox/style.ts

```diff
@@ -168,19 +168,21 @@ export default class StyleSandbox extends SandboxBase {
         return proxyObject;
     }
 
-    _overrideCSSStyleDeclarationFunctionsCtx (window: Window) {
-        // @ts-ignore
+    _overrideCSSStyleDeclarationFunctionsCtx (window: Window & typeof globalThis) {
         const styleDeclarationProto = window.CSSStyleDeclaration.prototype;
 
         for (const prop in styleDeclarationProto) {
-            // @ts-ignore
             const nativeFn = this.nativeMethods.objectGetOwnPropertyDescriptor.call(window.Object, styleDeclarationProto, prop).value;// eslint-disable-line no-restricted-properties
 
             if (this.nativeMethods.objectHasOwnProperty.call(styleDeclarationProto, prop) &&
                 typeof nativeFn === 'function') {
-                styleDeclarationProto[prop] = function () {
+                (styleDeclarationProto[prop] as unknown as Function) = function () {
                     return nativeFn.apply(this[CSS_STYLE_PROXY_TARGET] || this, arguments);
                 };
+
+                // NOTE:
```

Comment:
Use the follwong comment style:

```js
// NOTE: we cannot use 'overrideFunction' here since the function may not exist

// TODO: this function should not be used outside this file
```